cs3api4lab/auth/eos.py

```diff
@@ -0,0 +1,38 @@
+import os
+from io import TextIOWrapper
+
+from jupyter_core.paths import jupyter_config_path
+from notebook.services.config import ConfigManager
+
+from cs3api4lab.auth.authenticator import Authenticator
+
+
+class Eos(Authenticator):
+    tokens = {}
+
+    def __init__(self, config=None, log=None):
```

Comment:
Since I said we should properly inherit from the parent, I would call the parent init and let if set the config.